packages/sdks/javascript/src/DevopnessApiClient.ts

```diff
@@ -90,23 +90,4 @@ export class DevopnessApiClient {
     this.users = new UserService();
     this.variables = new VariableService();
   }
-
-  public get accessToken(): string {
-    return ApiBaseService.accessToken;
-  }
-
-  public set accessToken(accessToken: string) {
-    const MIN_TOKEN_LENGHT = 100;
-
-    /**
-     * As a complete access token validation rule might be too much to be implemented here, we
-     * at least check for min length and return a substring of it to help users identify the
-     * issue when first initializing this SDK from their apps
-     */
-    if (accessToken && accessToken.length < MIN_TOKEN_LENGHT) {
-      throw new Error(`"${accessToken.substring(0, 10)} ..." doesn't seem to be a valid access token issued by Devopness API.`);
-    }
-
-    ApiBaseService.accessToken = accessToken;
-  }
```

Comment:
The removed code was applying at least minimum validation on the received `accessToken`.
If you believe it's not useful/no longer needed, could you please make sure of mentioning in the PR description the reason why it's beem removed?

Reply:
Yes, sure! About why it was removed, please help me also understand, I asked it [here](https://github.com/devopness/devopness/pull/724#discussion_r1276540367)